copr_script.sh

```diff
@@ -0,0 +1,49 @@
+#! /bin/bash -x
+
+# Preset variables
+readonly SPEC_FILE="hardened-chromium-subresource-filter.spec"
+readonly SPEC_SHA384_CHECKSUM="3a75b3b1605e11b8c084775ab92efddcb3e106c441d176ff62c3c4b546b04181ad04a60b1d251b84a101dd9830fc7d9b"
+readonly GIT_URL="https://raw.githubusercontent.com/secureblue/hardened-chromium-subresource-filter/refs/heads/"
```

Comment:
this errors out, also, im pretty sure git already does checksumming. we should just clone the entire repo:

https://github.com/secureblue/hardened-chromium-subresource-filter.git

Reply:
@RKNF404 not sure what you mean, shouldn't we be checksumming the easylists and not our own spec?

Reply:
ah okay scratch that then